webpack/ForemanInventoryUpload/Components/PageHeader/components/CloudPingModal/index.js

```diff
@@ -2,7 +2,11 @@
 import React, { useCallback, useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
-import { Table, TableBody, TableHeader } from '@patternfly/react-table';
+import {
+  Table,
+  TableBody,
+  TableHeader,
+} from '@patternfly/react-table/deprecated';
```

Comment:
Do we have migration steps to use a non-deprecated elements?

Reply:
We have these docs: https://www.patternfly.org/get-started/upgrade#upgrade-deprecated-components